packages/client/modules/email/components/EmailNotifications/EmailNotificationPicker.tsx

```diff
@@ -0,0 +1,57 @@
+import graphql from 'babel-plugin-relay/macro'
+import {ValueOf} from 'parabol-client/types/generics'
+import {EmailNotificationPicker_notification$key} from 'parabol-client/__generated__/EmailNotificationPicker_notification.graphql'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import EmailKickedOut from './EmailKickedOut'
+import EmailMeetingStageTimeLimitEnd from './EmailMeetingStageTimeLimitEnd'
+import EmailResponseMentioned from './EmailResponseMentioned'
+import EmailTeamArchived from './EmailTeamArchived'
+import EmailTeamInvitation from './EmailTeamInvitation'
+
+export const typePicker = {
+  KICKED_OUT: EmailKickedOut,
+  // PAYMENT_REJECTED: EmailPaymentRejected,
+  // TASK_INVOLVES: EmailTaskInvolves,
+  // PROMOTE_TO_BILLING_LEADER: EmailPromoteToBillingLeader,
```

Comment:
+1 is that something we'll add soonish? if yes, let's add a comment. If not, it's better to remove commented stuff

Reply:
Added a TODO. I'll be implementing these in a follow-up PR (this PR was getting big).